e2e-tests/portal/cypress/e2e/journeys/admin/restrict-access/restrict-access.spec.js

```diff
@@ -10,20 +9,14 @@ context('Only allow authorised users to access admin pages', () => {
   });
 
   describe('Access a deal', () => {
-    let deal;
-
     before(() => {
-      const aDealInStatus = (status) => deals.filter((aDeal) => status === aDeal.status)[0];
-
       cy.deleteDeals(ADMIN);
-      cy.insertOneDeal(aDealInStatus('Draft'), BANK1_MAKER1).then((insertedDeal) => {
-        deal = insertedDeal;
-      });
+      cy.createBssEwcsDeal({});
     });
 
     it('allows read only user with all bank access to view deal', () => {
-      cy.loginGoToDealPage(ADMIN, deal);
-      cy.url().should('eq', relative(`/contract/${deal._id}`));
+      cy.loginGoToDealPage(ADMIN);
+      cy.url().should('include', '/contract');
```

Comment:
why are we not strictly checking this url now?

Reply:
We can't get the deal id when creating deals through UI. @ttbarnes suggested to use `/contract`. @ttbarnes please suggest if this is correct.

Reply:
I would like to refactor a lot of the tests - I think instead of just lose checking we should use `getDealIdFromUrl` to get the deal ID and stricty assert

Reply:
To be checked.

Reply:
Used getDealIdFromUrl to get deal ID